tsconfig.json

```diff
@@ -13,7 +13,6 @@
         "declaration": true,
         "jsx": "react",
         "lib": ["es2021", "dom", "dom.iterable"],
-<<<<<<< HEAD
         "strict": true,
```

Comment:
for next time : if the trailing comma is what created this conflict, we can move the tchap code up so that it's not the last element in the json object